docs/install/Knative-custom-install.md

```diff
@@ -241,10 +245,15 @@ for details about installing the various supported observability plug-ins.
   https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin.yaml
 [1.6]:
   https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin-in-mem.yaml
+[1.7]:
```

Comment:
Its not super important to organize this list too (since its main goal was to allow readability in the markdown tables). 

BUT, if you are up for it, its more intuitive for the next contributor who touches this file for all the links and their reference #s to be in order too. Totally okay to skip this. 

I think for long term maintenance, all the numbers should be reordered and include gaps to allow for easier insertion for future items (and thats a bit more work and out of scope of this PR)